js_modules/dagit/packages/core/src/runs/RunsRoot.tsx

```diff
@@ -101,119 +101,121 @@ export const RunsRoot: React.FC<RouteComponentProps> = () => {
+          <Box
+            border={{side: 'bottom', width: 1, color: Colors.LIGHT_GRAY3}}
+            flex={{direction: 'row', justifyContent: 'space-between', alignItems: 'flex-end'}}
+          >
+            <Tabs selectedTabId={selectedTab} id="run-tabs">
+              <Tab
+                title={
+                  <TabButton
+                    color={tabColor('all')}
+                    underline="never"
+                    onClick={() => setStatusFilter([])}
+                  >
+                    All runs
+                  </TabButton>
+                }
+                id="all"
+              />
+              <Tab
+                title={
+                  <TabButton
+                    color={tabColor('queued')}
+                    underline="never"
+                    onClick={() => setStatusFilter(Array.from(queuedStatuses))}
+                  >
+                    <Group direction="row" spacing={4} alignItems="center">
+                      <div>Queued</div>
+                      <CountTag
+                        loading={queryResult.loading && !queryResult.data}
+                        fragment={
+                          queryResult.data?.queuedCount?.__typename === 'PipelineRuns'
+                            ? queryResult.data?.queuedCount
+                            : undefined
+                        }
+                      />
+                    </Group>
+                  </TabButton>
+                }
+                id="queued"
+              />
+              <Tab
+                title={
+                  <TabButton
+                    color={tabColor('in-progress')}
+                    underline="never"
+                    onClick={() => setStatusFilter(Array.from(inProgressStatuses))}
+                  >
+                    <Group direction="row" spacing={4} alignItems="center">
+                      <div>In progress</div>
+                      <CountTag
+                        loading={queryResult.loading && !queryResult.data}
+                        fragment={
+                          queryResult.data?.inProgressCount?.__typename === 'PipelineRuns'
+                            ? queryResult.data?.inProgressCount
+                            : undefined
+                        }
+                      />
+                    </Group>
+                  </TabButton>
+                }
+                id="in-progress"
+              />
+              <Tab
+                title={
+                  <TabButton
+                    color={tabColor('done')}
+                    underline="never"
+                    onClick={() => setStatusFilter(Array.from(doneStatuses))}
+                  >
+                    Done
+                  </TabButton>
+                }
+                id="done"
+              />
+              <div style={{display: 'flex', alignSelf: 'stretch'}}>
+                <Divider style={{margin: '6px 0px'}} />
+              </div>
+              <Tab
```

Comment:
If we upgrade this to use our new TabsWIP, I think that the tabs can now have `onClick` and this doesn't need to be a tab with a button inside anymore...